pkg/cmd/server/api/types.go

```diff
@@ -346,6 +346,10 @@ type ServiceAccountConfig struct {
 	// Each key is tried in order until the list is exhausted or verification succeeds.
 	// If no keys are specified, no service account authentication will be available.
 	PublicKeyFiles []string
+
+	// MasterCA is the CA for verifying the TLS connection back to the master.  The service account controller will automatically
+	// inject the contents of this file into pods so they can verify connections to the master.
+	MasterCA string
```

Comment:
> do we want this field replicated in several places, or a single top-level field?

I would replicate it so that the individual configs stay complete in and of themselves, but I don't feel very strongly.